ansible/roles/jitsi-meet-prosody/tasks/main.yml

```diff
@@ -28,8 +28,14 @@
   args:
     chdir: "{{ jitsi_meet_prosody_tempdir.path }}"
 
+- name: Ensure prosody plugins dir
+  ansible.builtin.file:
+    path: /usr/share/jitsi-meet/prosody-plugins
+    state: directory
+    mode: 0755
+
 - name: Copy jitsi-meet-prosody plugins to appropriate location
-  ansible.builtin.command: cp -a ./usr/share/jitsi-meet/prosody-plugins/ "/usr/share/jitsi-meet/prosody-plugins"
+  ansible.builtin.command: cp -a ./usr/share/jitsi-meet/prosody-plugins/* /usr/share/jitsi-meet/prosody-plugins
```

Comment:
I was testing this with the param and it was working. 
Before it was `cp -a ./usr/share/jitsi-meet/prosody-plugins /usr/share/jitsi-meet`